frontend/catalyst/pennylane_extensions.py

```diff
@@ -465,6 +467,95 @@ def _check(x, hint):
     return jprim.vjp_p.bind(*params, *cotangents, jaxpr=jaxpr, fn=fn, grad_params=grad_params)
 
 
+class Adjoint(Operation):
+    """A minimal implementation of PennyLane operation, designed with a sole purpose of being
+    placed on the quantum tape"""
+
+    num_wires = AnyWires
+
+    def __init__(self, body_jaxpr, consts_jaxpr, cargs_jaxpr):
+        self.body_jaxpr = body_jaxpr
+        self.consts_jaxpr = list(consts_jaxpr)
+        self.cargs_jaxpr = list(cargs_jaxpr)
+        super().__init__(wires=Wires(Adjoint.num_wires))
+
+
+def adjoint(f: Union[Callable, Operator]) -> Union[Callable, Operator]:
```

Comment:
@grwlf the adjoint function is not appearing in the API documentation (see https://xanaduai-pennylane--138.com.readthedocs.build/projects/catalyst/en/138/code/__init__.html).

This is because you will need to add it to the `__all__` variable in `__init__.py`. Once done, you can check the build above to make sure the docstring is rendering correctly.

Reply:
Thanks, indeed!

Reply:
I also added missing jvp/vjp functions!